test/commands/force/apex/test/run.test.ts

```diff
@@ -28,6 +22,9 @@ import {
   runWithFailures,
   testRunSimple
 } from './testData';
+import { tmpdir } from 'os';
+
+const outDir = tmpdir();
```

Comment:
Importing tmpdir and using it as the --outputdir value in the three tests below that were using 'my/path/to/dir' got the Windows unit tests passing on the GHA build.